plugins/modules/system/dconf.py

```diff
@@ -189,6 +186,7 @@ def _get_existing_dbus_session(self):
                 if process_real_uid == uid and 'DBUS_SESSION_BUS_ADDRESS' in process.environ():
                     dbus_session_bus_address_candidate = process.environ()['DBUS_SESSION_BUS_ADDRESS']
                     self.module.debug("Found D-Bus user session candidate at address: %s" % dbus_session_bus_address_candidate)
+                    self.module.get_bin_path('dbus-send', required=True)
```

Comment:
Why not re-use the path in the command?

Reply:
How about moving this into the `__init__` function check next to the check for `dbus-run-session` command instead?